EmpleoDotNet/Views/Shared/_FindJobsPartial.cshtml

```diff
@@ -133,18 +105,15 @@
                         <span class="search-job-label-item"><a href="@keyword.Link">@keyword.Description</a></span>
                     }
                 </p>
-                @*<p class="search-job-label">
-                    <strong>Localidades :</strong>
-                    @foreach (var location in locations)
-                    {
-                        <span class="search-job-label-item"><a href="@location.Link">@location.Description</a></span>
-                    }
-                </p>*@
```

Comment:
Como buenos boys and girl scouts vamos a quitar todos los comments que no hacen nada, como este